docs/application/vscode-ext/web.md

```diff
@@ -79,142 +50,156 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
+> If you have a Tizen device and you want to build the application into it, you must first generate, and then add a certificate.
 
 ### Deploy and Run Application in Emulator
 
 To deploy and run application, follow these steps:
 
-1. Open **Command Palette** and enter **Run Emulator-Manager**. The Tizen Emulator Manager is launched.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Run Emulator-Manager**, and press **Enter**.
 
    > **Note**  
    >
-   > You cannot launch an emulator directly from VS Code. Instead, you must launch the Tizen Emulator Manager and use it to launch the emulator you need.
+   > To launch an emulator in VS Code, you must launch Tizen Emulator Manager and select the emulator you want to launch.
 
-2. Create and launch an emulator instance in the Emulator Manager.
-3. To deploy your application to the target, enter **Install Tizen App** in the **Command Palette**.
-4. To run the application on the emulator, enter **Run Tizen App** in the **Command Palette**.
+3. Create and launch an emulator instance in the Emulator Manager.
+4. To deploy your application to the target, type **Install Tizen App** in the **Command Palette**.
+5. To run the application on the emulator, type **Run Tizen App** in the **Command Palette**.
 
-To install your project:
+To install project:
 
-  1. Open **Command Palette** and enter **Install Tizen App**.
-  2. Enter the folder name of the project you want to install. (Folder name, not path)
-  3. When the installation is successfully completed, the installation app is generated. Also, you will see a build success log in the **terminal** at the bottom of the vscode.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Install Tizen App**, and press **Enter**.
+3. Enter the folder name of the project you want to install. (Folder name, not path)
+4. When the installation is successfully completed, the installation app is generated. Also, you will see a build success log in the **terminal** at the bottom of the vscode.
 
   ![Install your tizen app](media/tizen_web14.png)
 
 > **Note** 
 > 
-> If you have a Tizen device and you want to deploy the application to it, you must first generate, and then add a certificate.
+> If you have a Tizen device and you want to deploy the application to it, then you must first generate a certificate and then add the certificate.
 
-To run your project:
+To run project:
 
   1. Open **Command Palette** and enter **Run Tizen App**.
-  2. Enter the folder name of the project you want to run. (Folder name, not path)
-  3. When the run is complete, it succeeds, and there is successful generation of launch app popup. Also, you will see a build success log in the terminal at the bottom of the vscode.
+  2. Enter the folder name of the project that you want to run. (Folder name, not path)
```

Comment:
Please remove "(Folder name, not path)" and add it as a Note:

Ensure that you enter the name of the folder and not the path.

Reply:
done.